src/pyvesync/helpers.py

```diff
@@ -276,3 +282,125 @@ def bypass_header():
             'Content-Type': 'application/json; charset=UTF-8',
             'User-Agent': 'okhttp/3.12.1',
         }
+
+    @staticmethod
+    def named_tuple_to_str(named_tuple: NamedTuple) -> str:
+        """Convert named tuple to string."""
+        tuple_str = ''
+        for key, val in named_tuple._asdict().items():
+            tuple_str += f'{key}: {val}, '
+        return tuple_str
+
+
+class HSV(NamedTuple):
+    """HSV color space."""
+
+    hue: int
```

Comment:
Why Int ? the api talks float on the `Hue` and `Saturation` parameters.
I noticed that same thing repeats through out the whole PR, everywhere as int.

Reply:
The HSV NamedTuple is never directly instantiated, it is just meant to hold the current color values, which always passed through the `Color` dataclass that does all the type validations. Color and the class methods that operate the color api accept `NUMERIC_T = Union[int, float, str]` . The api will receive a float when the int is implicitly converted as it is operated on by the float factors. So externally, `bulb.set_hsv(100, "53", 45.0) == bulb.set_hsv("100", 53.0, 45)`

Reply:
I get that the type union, and validators, makes it accept the other types, which is great!!, but also, ultimately, everything is being converted to an Int in the process, so a value of 230.4 is being rounded to 230.
Since the final api call will accept integers in the range of 0-10000, that missing 0.4 does make a difference.

Reply:
I didn't realize the 0.4 makes a difference, I'll change the internal type to float. That's an easy fix